### PR TITLE
feat(frontend): add price in register page

### PR DESCRIPTION
**Pull request by zuramai**

### CODE REVIEW

File: frontend/pages/register.vue [hunk 1]
```diff
@@ -53,6 +53,9 @@ const submit = async () => {
                 <Btn size="lg">Save my spot</Btn>
             </form>
 
+            <p class="mb-0!">Price: <span class="text-white glow">Rp 100.000</span> / seat</p>
+            <p>*Students get 25K discount </p>
```
Comment (aldy505) [inline frontend/pages/register.vue:57]:
```suggestion
            <p>*Students can get Rp25.000 discount, terms and condition applies</p>
```

**Review by aldy505 — APPROVED** [review]

File: frontend/pages/register.vue (hunk 1 above)
Comment (WahidinAji) [reply frontend/pages/register.vue:57]:
```suggestion
           <p>*Students can get a 25% discount or IDR 25,000, terms and conditions applies</p>
```

**Review by WahidinAji — CHANGES_REQUESTED** [review]
a little bit of correction

File: frontend/pages/register.vue (hunk 1 above)
Comment (WahidinAji) [reply frontend/pages/register.vue:57]: @aldy505

**Pull request merged**